libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts

```diff
@@ -49,6 +49,7 @@ export class RecordedDashboards implements IWorkspaceDashboardsService {
                 created: dashboard.created,
                 description: dashboard.description,
                 updated: dashboard.updated,
+                tags: dashboard.tags,
```

Comment:
this probably needs ?? [] for existing recordings?

Reply:
Good point, updated.